Homework-№1-from-07.09.2022/task6.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+#define maxSize 1000
+
+void creation(char *array, int lenOfArray)
+{
+    for (int i = 0; i < lenOfArray; ++i)
+    {
+        array[i] = ' ';
+    }
+}
+
+int lenOfString(char *string, int lenOfString)
+{
+    int trueLen = 0;
+    for (int i = 0; i < lenOfString; ++i)
+    {
+        if (string[i] != ' ')
+        {
+            trueLen++;
+        }
+    }
+    return trueLen;
+}
+
+int task6(char *firstStr, char *secondStr)
+{
+
+    int isItSubstring = 1, answer = 0;
+    int lenFirstStr = lenOfString(firstStr, maxSize), lenSecondStr = lenOfString(secondStr, maxSize);
+    if (lenFirstStr < lenSecondStr)
+    {
+        return -1;
+    }
+    else {
+        for (int symbolOfFirstStr = 0; symbolOfFirstStr < lenFirstStr - lenSecondStr + 1; symbolOfFirstStr++)
+        {
+            isItSubstring = 1;
+            for (int symbolOfSecondStr = 0; symbolOfSecondStr < lenSecondStr; symbolOfSecondStr++)
+            {
+                if (firstStr[symbolOfFirstStr + symbolOfSecondStr] != secondStr[symbolOfSecondStr])
+                {
+                    isItSubstring = 0;
+                    break;
+                }
+            }
+            answer += isItSubstring;
+        }
+        return answer;
+    }
+}
+
+int main()
+{
+    char firstStr[maxSize], secondStr[maxSize];

```

Comment:
```suggestion
    char firstStr[maxSize] = { 0 }, secondStr[maxSize] = { 0 };
```
Этого достаточно для инициализации строк нулевыми символами.